core/variant/variant.h

```diff
@@ -31,7 +31,6 @@
 #ifndef VARIANT_H
 #define VARIANT_H
 
-#include "core/input/input_enums.h"
```

Comment:
This isn't valid, you can't forward declare these enums they're used

Reply:
Why, they are not used anywhere, and if they are, they are already included in the file that uses them

Reply:
Read the source of the header:
https://github.com/godotengine/godot/blob/fe01776f05b1787b28b4a270d53037a3c25f4ca2/core/variant/variant.h#L494-L507